articles/active-directory-b2c/partner-nok-nok.md

```diff
@@ -80,57 +82,47 @@ For the following instructions, use the directory with the Azure AD B2C tenant.
 2. For **Name**, enter the Nok Nok Authentication Provider, or another name.
-3. For **Metadata URL**, enter the hosted Nok Nok Authentication app URI, followed by the path such as `https://demo.noknok.com/mytenant/oidc/.well-known/openid-configuration`.
+3. For **Metadata URL**, enter the following url after replacing the placeholder with the tenant id provided by Nok Nok: `https://cloud.noknok.com/<tenant_id>/webapps/nnlfed/realms/<tenant_id>/.well-known/openid-configuration`.
 4. For **Client Secret**, use the Client Secret from Nok Nok.
 5. For **Client ID**, use the Client ID provided by Nok Nok.
-6. For **Scope**, use **OpenID profile email**.
+6. For **Scope**, use **openid**.
 7. For **Response type**, use **code**.
 8. For **Response mode**, use **form_post**.
-9. Select **OK**.
-10. Select **Map this identity provider’s claims**.
-11. For **UserID**, select **From subscription**.
-12. For **Display name**, select **From subscription**.
-13. For **Response mode**, select **From subscription**. 
-14. Select **Save**.
+9. For **User ID**, use **sub**.
+10. For **Display name**, use **sub**.
+11. Select **Save**.
 
 ### Create a user flow policy
 
 For the following instructions, Nok Nok is a new OIDC IdP in the B2C identity providers list.
 
-1. In your Azure AD B2C tenant, under **Policies**, select **User flows**.
+1. In your AAD B2C tenant, under **Policies**, select **User flows**.
 2. Select **New**.
 3. Select **Sign up and sign in**.
-4. Select a **version**.
+4. Select a **Version**.
 5. Select **Create**.
 6. Enter a policy **Name**.
 7. In **Identity providers**, select the created Nok Nok IdP.
-8. You can add an email address. Azure won't redirect sign-in to Nok Nok; a screen appears with user options.
+8. Check **Email signup** under Local accounts to display an intermediate AAD B2C signin/signup page with a button that redirects the user to the Nok Nok Signin App.
 9. Leave the **Multi-factor Authentication** field.
-10. Select **Enforce conditional access policies**.
-11. Under **User attributes and token claims**, in the Collect attribute option, select **Email Address**. 
-12. Add user attributes for Microsoft Entra ID to collect, with claims that Azure AD B2C returns to the client application.
-13. Select **Create**.
-14. Select the new **User flow**.
-15. On the left panel, select **Application Claims**. 
-16. Under options, select the **email** checkbox
-17. Select **Save**.
+10. Click on **Create** to save.
 
 ## Test the user flow
 
-1. Open the Azure AD B2C tenant. Under **Policies** select **Identity Experience Framework**.
+1. Open the AAD B2C tenant. Under **Policies** select **Identity Experience Framework**.
 2. Select the created **SignUpSignIn**.
 3. Select **Run user flow**.
 4. For **Application**, select the registered app. The example is JWT.
-5. For **Reply URL**, select the redirect URL.
+5. For **Reply URL**, select the redirect URL of the application selected at the previous step.

```

Comment:
Suggestion to change passive voice to active voice: change "selected" to "that you selected"